ddtrace/tracer/tracer.go

```diff
@@ -162,6 +162,7 @@ func newUnstartedTracer(opts ...StartOption) *tracer {
 func newTracer(opts ...StartOption) *tracer {
 	t := newUnstartedTracer(opts...)
 	c := t.config
+	log.Debug("Tracer starting. %#v", c)
```

Comment:
I think we can remove this log. We already have startup logging [here](https://github.com/DataDog/dd-trace-go/blob/v1/ddtrace/tracer/tracer.go#L97).